test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo14-addon-stock_partner_delivery_window @git+https://github.com/OCA/stock-logistics-workflow@refs/pull/940/head#subdirectory=setup/stock_partner_delivery_window
```

Comment:
A good habit is to keep PRs in Draft state while there are dependencies on unmerged addons. This avoids merging by accident.